components/nimbus/examples/experiment.rs

```diff
@@ -282,8 +283,12 @@ fn main() -> Result<()> {
             'outer: loop {
                 let uuid = uuid::Uuid::new_v4();
                 let mut num_of_experiments_enrolled = 0;
+                let targeting_attributes = TargetingAttributes {
+                    app_context: context.clone(),
+                    ..Default::default()
+                };
```

Comment:
Would this be less expensive if it was done outside of `loop`?

Reply:
Oop! good catch 😄